1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/InsertionSort.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
```

Comment:
Check this code

Sorted List not print

Reply:
But Sir.. It's Working....
![Screenshot_2021-06-21_19_45_03](https://user-images.githubusercontent.com/76242275/122777526-a4b4ac80-d29b-11eb-95b1-653705ce964b.png)
**It's Giving Output..**

Reply:
> But Sir.. It's Working....
> ![Screenshot_2021-06-21_19_45_03](https://user-images.githubusercontent.com/76242275/122777526-a4b4ac80-d29b-11eb-95b1-653705ce964b.png)
> **It's Giving Output..**

Ok, I run on an online compiler
So,  problem raise on online compiler